_data/engine-cli/docker_run.yaml

```diff
@@ -1353,13 +1353,13 @@ examples: |-
 
   ```console
   $ docker run --device=/dev/sdc:/dev/xvdc \
-               --device=/dev/sdd --device=/dev/zero:/dev/nulo \
+               --device=/dev/sdd --device=/dev/zero:/dev/null \
```

Comment:
Thanks for contributing! The files in this directory are generated from source code and markdown files in the https://github.com/docker/cli repository, and should not be manually edited.

That said, this also looks similar to https://github.com/docker/cli/pull/2451. Changing to `/dev/null` is not correct here, but the `nulo` name is a bit poorly chosen and confusing.

Let me open a quick pull request in the CLI repository to update it.